source/custom/nixnet_service.custom.cpp

```diff
@@ -204,6 +207,10 @@ ::grpc::Status NiXnetService::ReadState(::grpc::ServerContext* context, const Re
 
 //---------------------------------------------------------------------
 //---------------------------------------------------------------------
+// WriteState API has an input of type void * called StateValue. There are different possible values for 
+// StateValue based on Based on the value of StateID passed in by user. We have defined complex message
```

Comment:
"based on" is mentioned twice here, can we correct this?

Reply:
Fixed the comment.